shaptools/shell.py

```diff
@@ -187,3 +191,25 @@ def remove_user(user, force=False, root_user=None, root_password=None, remote_ho
         else:
             break
     raise ShellError('error removing user {}'.format(user))
+
+def extract_sapcar_cmd(sar_file, options=None, user=None, password=None, remote_host=None):
+    """
+    Execute SAPCAR command to decompress a SAP CAR or SAR files.
+    If user and password are provided it will be executed with this user.
+
+    Args:
+        sar_file (str): Path to the sar file to be extracted
+        options(str, opt): Additional options to SAPCAR command
+        user (str, opt): User to execute the SAPCAR command
+        password (str, opt): User password
+    """
+    if not os.path.isfile(sar_file):
+        raise FileDoesNotExistError('The SAR file \'{}\' does not exist'.format(sar_file))
+    if options is None:
+        cmd = './SAPCAR -xvf {sar_file}'.format(sar_file=sar_file)
```

Comment:
I would have the option `-manifest SIGNATURE.SMF` enabled by default as it generates the manifest used by the installer. Without it the installation fails.

Reply:
Is this always the case? I wouldn't like to have hardcoded things here. What if you don't want to use this option?
I would let this as it is, and set this option in higher levels (salt code for example)

Reply:
as discussed here, I have set this option from formula level as `options` parameter